src/core/Akka.Remote/PhiAccrualFailureDetector.cs

```diff
@@ -222,11 +222,14 @@ internal double Phi(long timestamp)
                 return 0.0d; //treat unmanaged connections, e.g. with zero heartbeats, as healthy connections
             else
             {
-                var timeDiff = timestamp - oldTimestamp.Value;
-                var history = oldState.History;
-                var mean = history.Mean;
-                var stdDeviation = EnsureValidStdDeviation(history.StdDeviation);
-                return Phi(timeDiff, mean + AcceptableHeartbeatPauseMillis, stdDeviation);
+                unchecked // in the extremely rare event of a clock roll-over
```

Comment:
Made this unchecked in the extremely rare event of a clock rollover. Realistically, this will never happen.